src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,18 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    String name;
+    ArrayList<Manager> managerList;
```

Comment:
Wiem wiem jeszcze o tym nie mówiliśmy na zajęciach ale ogólnie dobrze korzystać z interfejsów przy deklarowaniu pól czyli `List<Manager> managerList` zamiast `ArrayList<Manager> managerList` a dopiero w konstruktorze przypisywać konkretną implementację czyli `this.managerList = new ArrayList<>();`.

Taki zabieg pozwoliłby nam na przykład na następujący konstruktor:
```
public CEO(String name, List<Manager> managerList) {
    this.name = name;
    this.managerList = managerList;
}
```
Wtedy użytkownik może przy korzystaniu z naszej klasy zapodać jakąkolwiek implementację interfejsu List np. LinkedList bądź ArrayList i wszystko będzie ładnie działało:)